recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb

```diff
@@ -15,7 +15,7 @@ do_install() {
    find "${XT_DIR_ABS_SHARED_BOOT_DOMD}" -iname 'Image*' -exec \
```

Comment:
please fix commit message:
s/varibale/variable
"so meet the changes" sounds strange ;)

Reply:
Done